config/proto/targets.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
+
+package hermes.targets
+
+option go_package = "github.com/googleinterns/hermes/targets/proto";
```

Comment:
Is it intentional to not include step224-2020 in the github path?

Reply:
I updated this path in a later commit included in this pull request.